scss/_variables.scss

```diff
@@ -260,6 +261,20 @@ $transition-fade:             opacity .15s linear !default;
 $transition-collapse:         height .35s ease !default;
 
 
+// Responsive font sizes
+//
+// RFS Configuration.
+
+$rfs-minimum-font-size: 1rem !default;
```

Comment:
The vendor file is an exact copy of the sass file, so that we can easily sync Bootstrap with RFS, but the configuration should be done in `_variables.scss`. That's the idea.

Reply:
I've just removed these, the default config shouldn't be part of the `_variables.scss` file.